packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect } from 'react';
+import { useIntl } from 'react-intl';
+import {
+  SettingsPageTitle,
+  useFocusWhenNavigate,
+  useNotification,
+  NoPermissions,
+  useRBAC,
+} from '@strapi/helper-plugin';
+import { HeaderLayout, ContentLayout } from '@strapi/parts/Layout';
+import { Main } from '@strapi/parts/Main';
+import { Button } from '@strapi/parts/Button';
+import AddIcon from '@strapi/icons/AddIcon';
+import { useQuery } from 'react-query';
+import { useLocation, useHistory } from 'react-router-dom';
+import qs from 'qs';
+import DynamicTable from './DynamicTable';
+import { axiosInstance } from '../../../../../core/utils';
+import adminPermissions from '../../../../../permissions';
+
+const ApiTokenListView = () => {
+  useFocusWhenNavigate();
+  const { formatMessage } = useIntl();
+  const toggleNotification = useNotification();
+  const {
+    allowedActions: { canCreate, canDelete, canUpdate, canRead },
+  } = useRBAC(adminPermissions.settings['api-tokens']);
+  const { search } = useLocation();
+  const { push } = useHistory();
+
+  useEffect(() => {
+    push({ search: qs.stringify({ sort: 'name:ASC' }, { encode: false }) });
+  }, [push]);
+
+  const createAction = () => {
+    if (!canCreate) {
+      return null;
+    }
+
+    return (
+      <Button data-testid="create-api-token-button" startIcon={<AddIcon />} size="L">
+        {formatMessage({
+          id: 'Settings.apiTokens.create',
+          defaultMessage: 'Add Entry',
+        })}
+      </Button>
+    );
+  };
+
+  const { data: apiTokens, status, isFetching } = useQuery(
+    ['api-tokens', search],
+    async () => {
+      const {
+        data: { data },
+      } = await axiosInstance.get(`/admin/api-tokens`);
+
+      return data;
+    },
+    {
+      enabled: !!canRead,
+      keepPreviousData: true,
+      retry: false,
+      staleTime: 1000 * 20,
```

Comment:
Let's keep the default for now 😛

Reply:
I copied for the `User` page. Should I remove everything but the `enabled`?